File: PHP/level-XXX.php
<?php
for($i = 1; $i <= 100; $i++)
    echo empty(to_fizz_buzz($i)) ? (string) $i : to_fizz_buzz($i), "\n";

function to_fizz_buzz($i) {
    return ($i % 3 == 0 ? 'Fizz' : '')
          .($i % 5 == 0 ? 'Buzz' : '');
}
?>

File: Java/tmp/level-XXX.java
public class Main {
    public static void main(String[] args) {
        for(int i = 1; i <= 100; i++) {
            System.out.println(to_fizz_buzz(i).isEmpty() ? String.valueOf(i) : to_fizz_buzz(i));
        }
    }

    private static String to_fizz_buzz(int i) {
        return (i % 3 == 0 ? "Fizz" : "")
             + (i % 5 == 0 ? "Buzz" : "");
    }
}
